python/test/pipeline_tests.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (C) 2019 Jorgen Dokken, Matthew Scroggs and Garth N. Wells
```

Comment:
What are these tests for? They doesn't seem to match pytest layout.

Reply:
I meant to delete that file (it's leftover from when I separated out some of the slower tests and is now unused)